editor/editor_help.cpp

```diff
@@ -3818,7 +3827,20 @@ void EditorHelpBitTooltip::show_tooltip(EditorHelpBit *p_help_bit, Control *p_ta
 	tooltip->add_child(p_help_bit);
 	p_target->get_viewport()->add_child(tooltip);
 	p_help_bit->update_content_height();
+	// When FLAG_POPUP is false, it prevents the editor from losing focus when displaying the tooltip.
+	// This way, clicks and double-clicks are still available outside the tooltip.
+	tooltip->set_flag(Window::FLAG_POPUP, false);
```

Comment:
As far as I remember, this flag makes hyperlinks in the tooltip unclickable.

Reply:
Effectively. It works on Windows, but not on Linux (Wayland). Is there another way to display a popup without getting the focus?

Reply:
I found a way to make the hyperlinks works in tooltips on Linux. That also fix the hyperlink in the Editor Settings tooltips.

I removed the faulty code in `display_server_x11` that was added by #69678 to fix when the mouse button release was done outside the Color Picker window. I was not able to reproduce the problem after my modifications, and from what I tested, it did not even pass into the removed code in that situation.